product-service/src/services/product.service.ts

```diff
@@ -0,0 +1,10 @@
+import { productsData } from '@mocks/productsData';
+import { Product } from '@models/product.model';
+
+export const getAll = (): Product[] | [] => {
+    return productsData;
+}
+
+export const getById = (productId: string): Product | undefined => {
+    return productsData.find((el) => el.id === productId);
+}
```

Comment:
suggestion: to create a separate class with a service that provides its methods to the stakeholders via an aggregation approach

Reply:
it's a suggestion